src/test/shared/utilities/vsCodeUtils.test.ts

```diff
@@ -85,122 +74,88 @@ describe('getChannelLogger', function() {
     let outputChannel: MockOutputChannel
     let channelLogger: ChannelLogger
 
-    const runEachTestCase = async (onRunTest: TestRunner) => {
-        for (const logLevel of logLevels) {
-            for (const testDataCase of testData) {
-                // Reset loggers for each test case
-                logger = new TestLogger()
-                outputChannel = new MockOutputChannel()
-                channelLogger = getChannelLogger(outputChannel, logger)
```

Comment:
The reason these tests were changed in this PR is that getChannelLogger no longer accepts a logger object, and this larger test was re-initializing objects on the above two lines within the for loops of the test.